opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -0,0 +1,337 @@
+		return self._default_labels
+
+	def set_default_labels(self, value):
+			self._default_labels = value
+
+	def on_register_view(self, view):
+		""" create metric descriptor for the registered view  """
+		if view:
+			self.create_metric_descriptor(view)
+
+	def emit(self, view_datas):
+		""" export data to Stackdriver Monitoring """
+		if view_datas:
+			self.handle_upload(view_datas)
+
+	def export(self, view_datas):
+		""" export data to transport class """
+		if view_datas:
+			pass #self.transport.export(view_datas) || Uncomment this when transport class gets merged to this branch
+
+	def handle_upload(self, view_datas):
+    	""" handle_upload handles uploading a slice of Data, 
+		as well as error handling. 
+		"""
+		if view_datas:
+			self.upload_stats(view_datas)
+		
+	def upload_stats(self, view_datas):
+		""" It receives an array of view_data object 
+			and create time series for each value
+		"""
+		for request in self.make_request(view_datas, 
+							MAX_TIME_SERIES_PER_UPLOAD):
+			self.client.create_time_series(request[CONS_NAME],
+											request[CONS_TIME_SERIES])
+
+	def make_request(self, view_datas, limit):
+		""" Create the data structure that will be 
+			sent to Stackdriver Monitoring
+		"""
+		requests = []
+		time_series = []
+
+		resource = self.options.resource
+		for view_data in view_datas:
+			series = monitoring_v3.types.TimeSeries()
+			series.metric.type = namespaced_view_name(view_data.view.name)
+
+			if not resource:
+    				series.resource.type = 'global'
+			else:
+				series.resource = resource
+
+			for tag_value, agg in view_data.tag_value_aggregation_map.items():
+				point = series.points.add()
+				if agg.aggregation_type is aggregation.Type.DISTRIBUTION:
+					agg_data = view_data.tag_value_aggregation_map.get(tag_value).aggregation_data
+					
+					point.value.distribution_value.count = agg_data.count_data
+					point.value.distribution_value.mean = agg_data.mean_data
+					point.value.distribution_value.sum_of_squared_deviation = agg_data.sum_of_sqd_deviations
+
+					# Uncomment this when stackdriver supports Range
+					# point.value.distribution_value.range.min = agg_data.min
+					# point.value.distribution_value.range.max = agg_data.max
+
+					point.value.distribution_value.bucket_options.explicit_buckets.bounds.extend(list(map(float,agg_data.bounds)))
+					point.value.distribution_value.bucket_counts.extend(list(map(int,agg_data.counts_per_bucket)))
+				elif type(tag_value.value) is int:
+					point.value.int64_value = int(tag_value.value)
+				elif type(tag_value.value) is float:
+					point.value.float_value = float(tag_value.value)
+				elif type(tag_value.value) is str:
+					point.value.string_value = str(tag_value.value)
+
+				start = datetime.datetime.strptime(view_data.start_time,"%Y-%m-%dT%H:%M:%S.%fZ")
+				end = datetime.datetime.strptime(view_data.end_time,"%Y-%m-%dT%H:%M:%S.%fZ")
+
+				timestamp_start = (start - datetime.datetime(1970, 1, 1)).total_seconds()
```

Comment:
I think we could create a static variable at the beginning of the class definition, say
```python
EPOCH_DATETIME = datetime.datetime(1970, 1, 1)

def ...:
    timestamp_star = (view_data.end_time - EPOCH_DATETIME).total_seconds()
```